docs/.vuepress/theme/layouts/404.vue

```diff
@@ -1,22 +1,116 @@
+          0 0 0 .232.863.631.631 0 0 0 .863-.231.632.632 0 0 0-.231-.863zM68.164 97.924a.633.633 0 0 0-.349.823.631.631
+          0 0 0 .823.349.63.63 0 0 0 .349-.822.633.633 0 0 0-.823-.35zM105.688 79.949a.631.631 0 0 0 .349-.822.633.633
+          0 0 0-.823-.35.633.633 0 0 0-.349.823.631.631 0 0 0 .823.349z"/>
+          <path class="h" d="M85.027 102.02a1.216 1.216 0 0 0-.328 1.689 1.216 1.216 0 0 0 1.689.329 1.218 1.218 0 0
+          0 .328-1.69 1.216 1.216 0 0 0-1.689-.328zM128.127 115.676a.632.632 0 0 0-.878-.17.631.631 0 0
+          0-.17.877.632.632 0 0 0 .877.171.633.633 0 0 0 .171-.878zM70.227 109.684a.633.633 0 0 0-.349.823.632.632
+          0 0 0 .823.349.63.63 0 0 0 .349-.822.633.633 0 0 0-.823-.35z"/>
+          <circle class="b" cx="1.441" cy="1.441" r="1.441" transform="translate(38.493 123.804)"/>
+          <circle class="b" cx="1.441" cy="1.441" r="1.441" transform="translate(47.761 125.455)"/>
+          <path class="b" d="M42.627 135.402h-.008a10.452 10.452 0 0 1-5.259-1.53.5.5 0 0 1-.166-.687.5.5 0 0 1
+          .687-.166 9.451 9.451 0 0 0 4.755 1.383.5.5 0 0 1 .491.508.5.5 0 0 1-.5.492zM52.445 138.895a10.446 10.446
+          0 0 1-3.813-.721.5.5 0 0 1-.283-.648.5.5 0 0 1 .648-.283 9.425 9.425 0 0 0 4.922.536.5.5 0 0 1 .573.416.5.5
+          0 0 1-.415.572 10.413 10.413 0 0 1-1.632.128z"/>
+          <path class="c" d="M1.989 88.498a12.736 12.736 0 0 0 3.223 5.428 12.8 12.8 0 0 0-3.464-11.739 12.756
+          12.756 0 0 0 .241 6.311z"/>
+          <path class="b" d="M6.849 84.076a20.132 20.132 0 0 0-1.643 9.849 20.242 20.242 0 0 0 7.79-17.723 20.178
+          20.178 0 0 0-6.147 7.874zM102.681 124.735a16.586 16.586 0 0 1-5.01.69 23.927 23.927 0 0
+          1-12.75-3.75c-5.57-3.55-12.37-11.28-13.12-27.91-1.4-30.71 20.17-90.26 21.08-92.78a1.513 1.513 0 0 1
+          1.93-.9 1.5 1.5 0 0 1 .89 1.93c-.22.61-22.28 61.5-20.91 91.61.55 12.15 4.61 20.97 11.74 25.52a20.986 20.986
+          0 0 0 7.52 2.93l.42.14a36.369 36.369 0 0 1 8.21 2.52z"/>
+          <path d="M-457.64 2820.342a29.754 29.754 0 0 1-12.26 3.938c-6.2.519-11.928-1.461-15.325-5.3a12 12 0 0
+          1-2.645-4.862 12.756 12.756 0 0 1-.112-5.962 7.507 7.507 0 0 1 3.18-4.491 8.088 8.088 0 0 0-.407 1.357 11.2
+          11.2 0 0 0 2.8 10.012c3.865 4.365 10.739 6.817 17.513 6.249a29 29 0 0 0 10.2-2.851 32.787 32.787 0 0 1-2.944
+          1.91z" transform="translate(595.527 -2647.565)" style="fill:#6da2ff"/>
+          <path class="b" d="M130.306 182.459a1.5 1.5 0 0 1-1.38.92 1.505 1.505 0 0 1-.58-.12 1.5 1.5 0 0
+          1-.8-1.97c4.21-9.97 1.72-19.1-2.02-24.31-2.13-2.97-4.78-4.79-6.73-4.66a3.126 3.126 0 0
+          0-1.33.36c-.01.01-.03.01-.04.02a37.69 37.69 0 0 0-4.92 2.26l.06-.24c1.1-3.38 3.12-5.2 6.01-5.39 3.07-.21
+          6.58 1.98 9.39 5.9 4.21 5.88 7.04 16.12 2.34 27.23z"/>
+        </svg>
+        <h1>Page not found (404)</h1>
+        <a :href="homeUrl">Go back to Handsontable docs ⟶</a>
```

Comment:
The words "docs" and "documentation" are of two different registers. So far, we used "documentation" everywhere, but I definitely prefer "docs" – let me know if I can change it to "docs" in other places as well.

Whatever we choose, we should stick to one throughout the docs (brand consistency etc.)

Reply:
Sounds reasonable to me. After changes:

<img width="618" alt="Screenshot 2022-10-14 at 09 21 52" src="https://user-images.githubusercontent.com/8048526/195788326-8c572c11-3624-4dd0-9e10-ef4f47b10403.png">